src/modules/fancyzones/lib/ZoneSet.cpp

```diff
@@ -15,9 +17,10 @@ using namespace FancyZonesUtils;
 namespace
 {
     constexpr int C_MULTIPLIER = 10000;
-    constexpr int MAX_ZONE_COUNT = 50;
 
     /*
+      constexpr int MAX_ZONE_COUNT = 50;
```

Comment:
is this really a limitation hardcoded somewhere (I expect FZ Editor) ?

Reply:
It is. Well, it was at the time this was added here. I'll check whether it's still there.

Reply:
There is this request to remove the limit or at least to increase it. 
https://github.com/microsoft/PowerToys/issues/7051
Not in this PR but something to keep in mind.

Reply:
Actually, as `predefinedPriorityGridLayouts` is filled with per-field initializations, it is self-explanatory enough, so I'll remove this comment.

Max zone count is 40, and it is defined here:
https://github.com/microsoft/PowerToys/blob/master/src/modules/fancyzones/editor/FancyZonesEditor/MainWindow.xaml.cs#L21

Reply:
possible to make this a shared constant between both?  Or can we add a comment in both to make sure we are aware they have to stay in sync?

Reply:
also unclear why there is this limit

Reply:
> possible to make this a shared constant between both?
No need for that anymore. Only Editor needs to know about that limitation. We rely on editor not to send more zones than that.

> also unclear why there is this limit
Not sure. It's there for some time now (from the beginning?) .. It's a single line change whenever we decide to make it.

Reply:
It was mainly for the editor not handling well large number of zones, the first release was only allowing 11 zones, later we increased it, and we can make it larger like 512, but we will have to make sure it doesn't break the Editor.